broker-cli/commands/buy.js

```diff
@@ -40,7 +40,7 @@ async function buy (args, opts, logger) {
   try {
     const client = new BrokerDaemonClient(rpcAddress)
     const blockOrderResult = await client.orderService.createBlockOrder(request)
-    logger.info(blockOrderResult)
+    logger.info(JSON.stringify(blockOrderResult))
```

Comment:
do we need to call `JSON.strinfigy` here? What's the behavior if we dont?

Reply:
@treygriffith I think this is what you mean. I made a comment about this earlier. 

If we simply pass the object (in this case `blockOrderResult`) to `logger` it _wont actually_ be valid json. We need to use `JSON.stringify` on the return values (stdout)

Reply:
I have split the json and non-json output to . not touch the non-json output for now. we can beautify the non-json results separately.